packages/editor/src/components/collab-sidebar/commentButtonToolbar.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Noting that some of the file names don't follow the established convention (kebab-case).